src/test/java/com/google/cloud/discotoproto3converter/DiscoToProto3ConverterAppTest.java

```diff
@@ -62,6 +63,140 @@ public void convert() throws IOException {
     assertEquals(baselineBody, actualBody);
   }
 
+  @Test
+  public void convertVersioned() throws IOException {
+    DiscoToProto3ConverterApp app = new DiscoToProto3ConverterApp();
+    Path prefix = Paths.get("google", "cloud", "compute", "v1small");
+    Path discoveryDocPath =
+        Paths.get("src", "test", "resources", prefix.toString(), "compute.v1small-versioned.json");
+    Path generatedFilePath =
+        Paths.get(outputDir.toString(), prefix.toString(), "compute-versioned.proto");
+
+    app.convert(
+        discoveryDocPath.toString(),
+        null,
+        generatedFilePath.toString(),
+        "",
+        "",
+        "https://cloud.google.com",
+        "false",
+        "true");
+
+    String actualBody = readFile(generatedFilePath);
+
+    Path baselineFilePath =
+        Paths.get(
+            "src", "test", "resources", prefix.toString(), "compute-versioned.proto.baseline");
+    String baselineBody = readFile(baselineFilePath);
+    System.out.printf(
+        "*** @Test:convertVersioned():\n*** Discovery path: %s\n*** Generated file: %s\n*** Baseline file: %s\n",
+        discoveryDocPath.toAbsolutePath(),
+        generatedFilePath.toAbsolutePath(),
+        baselineFilePath.toAbsolutePath());
+
+    assertEquals(baselineBody, actualBody);
+  }
+
+  @Test
+  public void convertVersionedInconsistent() throws IOException {
```

Comment:
nit: Add a comment under each test which explains the purpose of each test

Reply:
Done.